profile-service/src/main/resources/db/migration/V9.14__update_reval_role_name.sql

```diff
@@ -0,0 +1,45 @@
+-- add the new roles
+
+INSERT INTO `Role` (`name`)
+VALUES
+      ('RevalApprover'),
+      ('RevalAdmin'),
+      ('RevalObserver')
+ON DUPLICATE KEY UPDATE `name` = `name`;
+
+-- update the user role with the new roles
+
+UPDATE `UserRole`
+SET `roleName` = 'RevalApprover'
+WHERE `roleName`='RevalTISAdmin1';
+
+UPDATE `UserRole`
+SET `roleName` = 'RevalAdmin'
+WHERE `roleName`='RevalTISAdmin2';
+
+UPDATE `UserRole`
+SET `roleName` = 'RevalObserver'
+WHERE `roleName`='RevalTISAdmin3';
+
+UPDATE `UserRole`
+SET `roleName` = 'HEE User Admin'
+WHERE `roleName`='RevalSiteAdmin';
+
+-- handle existing super admin user, it is a combination of HEE User Admin and Reval Approver
+
+DELETE
+FROM `UserRole`
+WHERE `roleName`
+IN ('RevalSuperAdmin');
```

Comment:
Further to the note on this page (https://hee-tis.atlassian.net/wiki/spaces/NTCS/pages/1671495738/Revalidation+-+Access+Permissions), `RevalSuperAdmin` is not required as it’s a combination of `HEE User Admin` and `Reval Approver`. Looking like we need to confirm the user will have both `HEE User Admin` and `Reval Approver` roles rather than just delete it.

Reply:
Further to the discussion with Ade, I agree we can `DELETE` this role. No `UPDATE` is needed. I will click the resolve button.